src/index.ts

```diff
@@ -38,6 +38,7 @@ export { default as QEdge } from './query_edge';
 export { default as QNode } from './query_node';
 export { default as InvalidQueryGraphError } from './exceptions/invalid_query_graph_error';
 export * from './qedge2apiedge';
+export {TrapiQueryGraph, TrapiResponse, TrapiResult};  // better readability
```

Comment:
What's the purpose of this? IIRC these are importable from `@biothings-explorer/types`

Reply:
Inferred tests was importing those from `index` [here](https://github.com/biothings/bte_trapi_query_graph_handler/actions/runs/10305390655/job/28526099732#step:7:354). I read that packages are re-exported from `index` for better readability, hence the changes. Instead, I can change the imports in tests.

Reply:
Yeah, the tests should be changed instead. That was a hold-over from when TRAPI types were defined in the query_graph_handler instead of the dedicated `types` package.

Reply:
This line should also be removed because no package should import TRAPI types from the `query_graph_handler` package